source/CardGames.py

```diff
@@ -432,6 +433,74 @@ def marshall_first_turn(self):
       # Print the board at the end of turn
       self.display_board_general()
 
+    def FugitiveRepeating(self):
+      draw = 0
+      NewHideouts = []
+      while draw != 1 and draw != 2 and draw != 3:
+        try:
+          draw = int(input("Select which deck to draw a card from ['1' (1-14), '2' (15-28), '3' (29-42)]: ").strip())
+        except:
+          ValueError
+      if draw == 1:
+        self.fugitive_deck.append(self.LowRangeDeck.pop())
+      elif draw == 2:
+        self.fugitive_deck.append(self.MidRangeDeck.pop())
+      elif draw == 3:
+        self.fugitive_deck.append(self.HighRangeDeck.pop())
+
+      card_value = self.fugitive_deck[-1].value
+      sprint_value = self.fugitive_deck[-1].sprint_value
+      print("You drew " + str(card_value) + " with a sprint value of " + str(sprint_value) + ".")
```

Comment:
This would be a big change but if you want to keep it consistent you should add the cards drawn for the fugitive to the fugitive Player class using the addCard function. Then, if you want, you could display the card ascii with the fugitive.showHand(-1) function and just 'say card drawn:'

I would have to tweak the drawHand() function to make it so that you can choose to display just the last card drawn but its like two line so it would be really easy and look good

Reply:
Got it. Georgio said we can meet and talk about how we are going to go about this so we will see what we decide.